website/docs/docs-section/roadmap/create-custom-context.md

```diff
@@ -4,4 +4,181 @@ title: Creating Custom Context
+  ],
+  "type": ["VerifiableCredential", "OpenAttestationCredential"],
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  "issuer": { ... },
+  "openAttestationMetadata": { ... },
+  "credentialSubject": {
+    "id": "id:example",
+    "date": "2018-02-21",
+    "customerId": "564",
+    "terms": "Due Upon Receipt",
+    "billFrom": {
+      "name": "ABC Company",
+      "streetAddress": "Level 1, Industry Offices",
+      "city": "Singapore",
+      "postalCode": "123456",
+      "phoneNumber": "60305029"
+    },
+    "billTo": {
+      "company": {
+        "name": "DEF Company",
+        "streetAddress": "Level 2, Industry Offices",
+        "city": "Singapore",
+        "postalCode": "612345",
+        "phoneNumber": "61204028"
+      },
+      "name": "James Lee",
+      "email": "def@company.com"
+    }
+    // other properties
+  }
+}
+```
+
+The `.json` above shows the structure of the `Invoice`'s raw document. The context created earlier, can be seen to be included in the `@context` field as a URL link.
+
+During the [wrapping](/docs/developer-section/libraries/open-attestation-cli#wrapping-documents) process, the document itself as well as the fields inside `credentialSubject` would be validated and cross checked against the contexts defined in `@context`.
+
+:::note
+It is not necessary that everything in `@context` has to appear in `credentialSubject`. But when there are additional items in `credentialSubject` that do not exist in what has already been defined in `@context`, an error would occur when trying to wrap the document.
+:::
+
+## Configuration File (Invoice)
+
+```json
+{
+  "network": "ropsten",
+  "wallet": {
+    "type": "ENCRYPTED_JSON",
+    "encryptedJson": "{\"address\":\"1245e5b64d785b25057f7438f715f4aa5d965733\",\"id\":\"bf069d1b-4e88-487c-b695-f2e03ed7c1ff\",\"version\":3, ...}"
+  },
+  "forms": [
+    {
+      "name": "Invoice",
+      "type": "VERIFIABLE_DOCUMENT",
+      "defaults": {
+        "version": "https://schema.openattestation.com/3.0/schema.json",
+        "@context": [
+          "https://www.w3.org/2018/credentials/v1",
+          "https://schemata.openattestation.com/io/tradetrust/invoice/1.0/invoice-context.json",
+          "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json"
+        ],
+        "type": ["VerifiableCredential", "OpenAttestationCredential"],
+        "issuanceDate": "2010-01-01T19:23:24Z",
+        "issuer": {
+          "id": "https://example.com",
+          "name": "DEMO STORE",
+          "type": "OpenAttestationIssuer"
+        },
+        "openAttestationMetadata": {...},
+        "credentialSubject": {}
+      },
+      "schema": {...}
+    }
+  ]
+}
+```
+
+The `.json` above shows the structure of the `Invoice`'s configuration file. Apart from the rest of the information such as `schema`, that contains custom fields to be filled in during the creation of the document, the `@context` could also be seen to be included.
```

Comment:
The part that says "the `@context` could also be seen to be included" is referring to the `@context` being included in the wrapped document after the wrapping process, is that right?

Reply:
i meant to say that the @context created in the top part is seen to be included inside this config file. But ya the @context will also be inside the wrapped document after wrapped and issued.